client/components/LearnerViewDirective.js

```diff
@@ -296,9 +296,9 @@ tie.directive('learnerView', [function() {
 
 
         $scope.questionIds = questionSet.getQuestionIds();
-        $scope.unlockedQuestions = [];
+        $scope.completedQuestions = [];
         for (var i = 0; i < $scope.questionIds.length; i++) {
-          $scope.unlockedQuestions.push(false);
+          $scope.completedQuestions.push(false);
```

Comment:
Is this just part of the initial setup, setting every question as "not completed?"

It might not be a bad idea to set a variable or add a comment here, just for future-proofing.